src/core/rendering.js

```diff
@@ -1158,23 +1158,25 @@ p5.prototype.clearDepth = function(depth) {
  * <div>
  * <code>
  * function setup() {
- *   createCanvas(100, 100);
+ *   // Create a canvas with WEBGL mode.
+ *   createCanvas(100, 100, WEBGL);
  *
+ *   // Set the background color.
  *   background(200);
  *
- *   // Set the blend mode.
+ *   // Set the blend mode to SUBTRACT.
  *   blendMode(SUBTRACT);
  *
  *   // Style the lines.
  *   strokeWeight(30);
  *
  *   // Draw the blue line.
  *   stroke('blue');
- *   line(25, 25, 75, 75);
+ *   line(-25, -25, 25, 25);  // Adjusted coordinates for WEBGL
```

Comment:
We probably don't need the "adjusted coordinates" comment, these are just the regular coordinates for a WebGL canvas.

Reply:
Ok done